interhost.py

```diff
@@ -8,8 +8,16 @@
 
 import Bio.AlignIO
 from Bio import SeqIO
-import argparse, logging, os, array, bisect
-try :
+import argparse, logging, os, sys, array, bisect
+
+# try to import izip (in the case of Python 2). 
+# No alternate needed as Python3 zip() is iterator-based.
+try:
+    from itertools import izip
+except ImportError:
+    pass
+
+try:
```

Comment:
How about merging the two try/import blocks?

```
try:
   from itertools import zip_longest
except ImportError :
   from itertools import izip_longest as zip_longest, izip as zip
```

Then farther down below, you can get rid of your sys.version_info thing (what if Python 4.0 comes around?) and just call "zip".  In the case of Python 2, the "from itertools import izip as zip" will overwrite "zip" in the local namespace with the itertools version.